Makefile

```diff
@@ -21,10 +21,15 @@ env/stop:
 env/teardown:
 	./bin/envteardown.sh
 
+doc/generate:
+	yarn run build:docs
+
 dev:
+	make doc/generate
```

Comment:
Why add that to the `dev` command? Same below for `make test`? I'm worried about the additional processing time.

Reply:
Good catch! I wanted to generate docs every time when running `make dev` (or `make test`). Regarding processing time, it seems we only need to run one time. Let me make the update.

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/89/commits/6391ef47b371a40433b500dd5c0775c31a735eca